ogc/src/main/scala/geotrellis/server/ogc/wcs/CoverageView.scala

```diff
@@ -68,6 +75,17 @@ object CoverageView {
     val ex = re.extent
     val llex = llre.extent
     val Dimensions(w, h) = re.dimensions
+    val temporalInstants: Seq[DataRecord[String]] = sources.flatMap { s =>
+      // TODO: Attempt to use `GmlDataRecord(TimePositionType(t.toWcsIsoString))`
+      //       fails with a "Missing element label" exception, not sure how this
+      //       doesn't have a CanWriteXML but all the other types do?
```

Comment:
Is it a question you have?

Reply:
Ah right, if it is a question, this is a weird thing about the protocol generated. 

Let me see what we can do here, I parsed the question.

Reply:
Sorta yes sorta no. It's a note to myself to resolve later, but if you have any suggestions in the meantime I'm all ears.

Reply:
@CloudNiner yep, it should always be the 

`TimeSequenceType(GmlDataRecord("timePosition", t.toWcsIsoString) :: GmlDataRecord("timePosition", t2.toWcsIsoString))` 

There is nothing to do with it; I think you're safe to remove this comment. 

I think types here are used for type safety only, but the key / name of the xml record lacks from the generated case classes. It leads to a neccesity of passing them manually. That's what I had to do with all other records as well.

Reply:
Ohhhh I see. I missed that its the `OwsDataRecord.apply(value)` that's constrcuting the element name from the passed class tag. 🎉